src/objects.js

```diff
@@ -1,41 +1,76 @@
 const createPerson = (name, age) => {
-  // your code here
+  // Create an object based on the name and age arguments
+  return { name, age };
 };
 
 const getName = object => {
-  // your code here
+  // Return the name property based on the object passed in
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  // Return object[property] based on the property String that's passed in
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  // Return true or false based on the hasOwnProperty method to check if property exists
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
+  // Return true or false is person.age is > than 65 using a ternary operator
+  return person.age > 65;
 };
 
 const getAges = people => {
-  // your code here
+  // Return new array with the person.age value using the map array method
+  return people.map(person => person.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  // Return object with person.name that matches name String that's passed in using array filter method
+  // 01
+  // return people.filter(person => person.name === name)[0];
+  // 02
+  return people.filter(person => person.name === name).pop();
 };
 
 const findHondas = cars => {
-  // your code here
+  // Return array of objects where car.manufacturer matches 'Honda' using array filter method
+  return cars.filter(car => car.manufacturer === 'Honda');
 };
 
 const averageAge = people => {
-  // your code here
+  // Return the average of the passed in people objects property using the array reduce method
+  // 01
+  // return people.reduce((sum, acc) => sum + acc.age, 0) / people.length;
```

Comment:
This method works, why is it commented out? The rest of the commented out stuff should be taken out here
```suggestion
  return people.reduce((sum, acc) => sum + acc.age, 0) / people.length;
```

Reply:
I think I was playing around with something late at night and might have forgot to uncomment the working code, doh!